src/lib/format.ts

```diff
@@ -3,10 +3,17 @@ const locale = 'en-US' // TODO: user preference
 const formatMoney = (
   amount: number,
   currency: string = '',
-  signDisplay: Intl.NumberFormatOptions['signDisplay'] = 'exceptZero'
+  options: {
+    signDisplay?: Intl.NumberFormatOptions['signDisplay']
+    hideZero?: boolean
+  } = {}
 ) => {
+  if (options.hideZero && amount == 0) {
```

Comment:
For typesafe comparison:
```suggestion
  if (options.hideZero && amount === 0) {
```